packages/components/src/unit-control/styles/unit-control-styles.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * External dependencies
+ */
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+/**
+ * Internal dependencies
+ */
+import { color, rtl } from '../../utils/style-mixins';
+import NumberControl from '../../number-control';
+
+export const Root = styled.div`
+	box-sizing: border-box;
+	position: relative;
+`;
+
+const fontSizeStyles = ( { size } ) => {
+	const sizes = {
+		default: null,
+		small: '11px',
+	};
+
+	const fontSize = sizes[ size ];
+
+	if ( ! fontSize ) return '';
+
+	return css`
+		@media ( min-width: 600px ) {
+			font-size: ${fontSize};
+		}
+	`;
+};
+
+const sizeStyles = ( { size } ) => {
+	const sizes = {
+		default: {
+			height: 30,
+			lineHeight: 30,
+			minHeight: 30,
+		},
+		small: {
+			height: 24,
+			lineHeight: 24,
+			minHeight: 24,
+		},
+	};
+
+	const style = sizes[ size ] || sizes.default;
+
+	return css( style );
+};
+
+// TODO: Resolve need to use &&& to increase specificity
+// https://github.com/WordPress/gutenberg/issues/18483
+
+export const ValueInput = styled( NumberControl )`
+	&&& {
+		appearance: none;
+		box-sizing: border-box;
+		border: 1px solid ${color( 'ui.border' )};
+		border-radius: 2px;
+		padding: 3px 8px;
+		display: block;
+		width: 100%;
+
+		&::-webkit-outer-spin-button,
+		&::-webkit-inner-spin-button {
+			-webkit-appearance: none;
+			margin: 0;
+		}
+
+		${rtl( { paddingRight: 20 } )}
+		${fontSizeStyles};
+		${sizeStyles};
+	}
+`;
+
+const unitSizeStyles = ( { size } ) => {
+	const sizes = {
+		default: {
+			top: 5,
+			height: 20,
+			minHeight: 20,
+		},
+		small: {
+			top: 4,
+			height: 16,
+			minHeight: 16,
+		},
+	};
+
+	return css( sizes[ size ] );
+};
+
+const baseUnitLabelStyles = ( props ) => {
+	return css`
+		appearance: none;
```

Comment:
I'm testing this on Firefox 75 in a Linux OS and the input control in Firefox includes the stepper arrows buttons:

![2020-04-13-191008-height-control-input-firefox](https://user-images.githubusercontent.com/583546/79141815-ccb4a000-7dba-11ea-9368-c6894869075a.png)

while in Chrome it doesn't:

![2020-04-13-191037-height-control-input-chrome](https://user-images.githubusercontent.com/583546/79141770-ba3a6680-7dba-11ea-8dbc-15d698310171.png)

For what is worth, I've tried with `-moz-appearance: textfield` and, as far as my testing went, it worked as expected (arrow up/down with the keyboard still increases/decreases the number) and the steppers are not shown.

cc @ItsJonQ @mcsf

Reply:
> I'm testing this on Firefox 75 in a Linux OS

Firefox (76) on macOS is also affected.

Reply:
Potential fix at https://github.com/WordPress/gutenberg/pull/21571